package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "vanilla-framework",
-  "version": "4.19.0",
+  "version": "4.19.1",
```

Version 4.19.0 has not yet been released. So this update can be ommited. Sorry for confusion.

```suggestion
  "version": "4.19.0",
```